pkg/antctl/raw/multicluster/join.go

```diff
@@ -217,13 +217,13 @@ func NewJoinCommand() *cobra.Command {
 	command.Flags().StringVarP(&joinOpts.LeaderNamespace, "leader-namespace", "", "", "Namespace of the leader cluster")
 	command.Flags().StringVarP(&joinOpts.LeaderClusterID, "leader-clusterid", "", "", "Cluster ID of the leader cluster")
 	command.Flags().StringVarP(&joinOpts.TokenSecretName, "token-secret-name", "", "", "Name of the Secret resource that contains the member token. "+
-		"Token Secret name takes precedence over token Secret file and the Secret manifest in the join config file")
+		"BearerToken Secret name takes precedence over token Secret file and the Secret manifest in the join config file")
```

Comment:
I think this is not an intended change?

Reply:
No, it is replaced by IDE automatically, thanks for catching it.